solr/contrib/ltr/src/java/org/apache/solr/ltr/response/transform/LTRFeatureLoggerTransformerFactory.java

```diff
@@ -187,8 +187,8 @@ public String getName() {
     }
 
     @Override
-    public void setContext(ResultContext context) {
```

Comment:
here is the problem: we have a plenty of transformers they are not changed?  (assuming they shouldn't be. see below)